File: 8_web-project-JSP/src/main/java/model/Project.java
package model;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;
import util.IdRandomGenerator;

@Data
@AllArgsConstructor
@NoArgsConstructor
public class Project {

    private int id;
    private String role;
    private String name;
    private int budget;
    private String email;
    private String password;

    public Project(String role, String name, int budget, String email, String password) {
        this.id = IdRandomGenerator.getRandomID();
        this.role = role;
        this.name = name;
        this.budget = budget;
        this.email = email;
        this.password = password;
    }
}

File: 8_web-project-JSP/src/main/webapp/js/project.js
let projects = null;

$.get("getProject", function (data) {
    if (data != '') {
        projects = data;
    }
}).done(function () {
    let cardsContent = '';

    jQuery.each(projects, function (i, project) {
        cardsContent += "<div class='card border-success mb-3' style='width: 350px; border-color: #004c40; margin-right: 3px; display: inline-block !important;'>" +
            "<div class='card-header'>" +
            "<h3>Project name : " + project.name + "</h3>" +
            "</div>" +
            "<div class='card-body text-success'>" +
            "<h4 style='color: #404E67;'>Project role : " + project.role + "</h4>" +
            "</div>" +
            "<div class='card-body text-success'>" +
            "<h4 style='color: #404E67;'>Project budget : " + project.budget + "</h4>" +
            "</div>" +
            "<div class='card-body text-success'>" +
            "<h4 style='color: #404E67;'>Project email : " + project.email + "</h4>" +
            "</div>" +
            "<div class='card-body text-success'>" +
            "<h4 style='color: #00796b;'>Project is bla bla bla</h4>" +
            "</div>" +
            "</div>"
    })

    $("div#projectCards").html(cardsContent);
});